app/mailers/attendance_mailer.rb

```diff
@@ -124,6 +124,8 @@ def mark_attendance_reminder(tea_time_id)
     @tea_time = TeaTime.find(tea_time_id)
     @host = @tea_time.host
```

Comment:
If you move this below cancel_delivery you'll save an extra read from the database because an exception will be thrown before the `tea_time.host` lookup is hit.